src/pages/signup/Signup.jsx

```diff
@@ -1,9 +1,56 @@
-import React from 'react';
-
+import React from "react";
+import { Link } from "react-router-dom";
 function Signup() {
-  return (
-    <div>Signup</div>
-  )
+	return (
+		<div className="login-form-container signup">
+			<form className="login_form-bookhub" action="">
+				<h3 className="heading-login">SIGN UP</h3>
+				<input
+					type="email"
+					name=""
+					className="box"
+					placeholder="Email"
+					aria-label="true"
+				/>
+				<input
+					type="password"
+					name=""
+					className="box"
+					placeholder="First name"
+					aria-label="true"
+				/>
+				<input
+					type="password"
+					name=""
+					className="box"
+					placeholder="Last name"
+					aria-label="true"
+				/>
+				<input
+					type="password"
+					name=""
+					className="box"
+					placeholder="Password"
+					aria-label="true"
+				/>
+				<input
+					type="password"
+					name=""
+					className="box"
+					placeholder="Confirm password"
+					aria-label="true"
+				/>
+				<Link to="/">
+					<input
+						type="submit"
+						value="sign up"
+						className="btn"
+						aria-label="true"
+					/>
+				</Link>
```

Comment:
Any reasons to use an input as a button?
It should be a ` button` element with `type="submit"`

Reply:
yes thank you👍